prime-select.sh

```diff
@@ -224,8 +224,8 @@ function common_set_intel {
     gpu_info=$(nvidia-xconfig --query-gpu-info)
     # This may easily fail, if no NVIDIA kernel module is available or alike
     if [ $? -ne 0 ]; then
-        logging "PCI BusID of NVIDIA card could not be detected!"
-        exit 1
+        logging "PCI BusID of NVIDIA card could not be detected! If service seems to work correctly you can ignore this message."
+        #exit 1 removed because service fails at boot if nvidia card is default off with bbswitch
```

Comment:
I don't like this approach since it will result in a broken xorg config file with an invalid line:

`BusID ""`

Reply:
If nvidia card is OFF  that section of xorg config file is useless, but if nvidia card isn't detected for other reasons it could be an issue. Probably different xorg config file(s) are needed for who uses bbswitch.